queues/c-or-cpp/queue-linked-list.cpp

```diff
@@ -0,0 +1,98 @@
+// Content: Queue using linked list (FIFO)
+// Author: Tawfik Yasser
+// Date: Fri, 5 Mar 2021
+
+#include <iostream>
+using namespace std;
+// Class `Node` to represent every item in the queue
+class Node
+{
+public:
+    int data;    // Item data
+    Node *next;  // Pointer to next item
+    Node() {}    // Empty constructor
+};
+// Claas to represent the queue using linked list
+class QueueLL
+{
+private:
+    Node *front; // Pointer to the head of the queue
+    Node *rear;  // Pointer to the end of the queue
+    int _length;
+public:
+    QueueLL() //Empty constructor to initialize front and rear to NULL because in the start the queue will be empty
+    {
+        front = rear = NULL;
```

Comment:
@TawfikYasser You have to initialize `_length` to `0` otherwise you'll get some garbage value. Also just a suggestion you should also print the size of the queue in the main function after some steps just so that the other reviewer's can easily verify that it's working correctly. Not saying that what you did is wrong but using `length` (or even `len`) and `size`  instead of `_length` and `_size` would be better in my opinion but it's upto you. Everything else works fine :)

Reply:
Okay @atin  Thanks I will do changes